modules/processing/static.py

```diff
@@ -322,6 +325,23 @@ def _get_signature(self):
 
         return ret
 
+    def _get_signature_verification(self):
+        """
+        Pulls the signature verification information for Authenticode.
+        :return: None if failure, otherwise a dict with the verification status and the output.
+        """
+        sig_path = os.path.join(self.analysis_path, "aux", "signtool.json")
+
+        if not os.path.exists(sig_path):
+            return None
+
+        with open(sig_path,'r') as sigin:
+            sigtxt = sigin.read()
+
+        sigdata = json.loads(sigtxt)
+
+        return sigdata
```

Comment:
This is where I was going to add more parsing at a later date, but I changed it how you wanted it for now.